### ISSUE TITLE
Image quality decreased drastically

### ISSUE THREAD

**Issue by SarenT** [syvaidya/openstego#17]
Hi,
I realized the output images of digital watermarking are much smaller than the input images with worse JPG quality. It would be great to preserve image quality. At least keeping JPG quality similar to input image.

Best,
SarenT

**Comment by syvaidya** [comment]
You can control the output compression for JPG by modifying openstego.ini file in user's home directory.

**Comment by SarenT** [comment]
I don't have that ini file in my user's home directory or elsewhere. I looked into the source code and it appears to be that one should have been created. I copied the one from the repo, which is supposed to be a template (it only includes that variable). My openstego ignored the file. My installation was from Ubuntu repositories. I cloned the most recent version now from github and built the project. The recent version created larger jpg files but they were all corrupted.
